lodash.js

```diff
@@ -3990,6 +3990,10 @@
         var key = toKey(path[index]),
             newValue = value;
 
+        if ((key === '__proto__' || key === 'constructor' || key === 'prototype')) {
```

Comment:
```suggestion
        if (key === '__proto__' || key === 'constructor' || key === 'prototype') {
```
Are the second set of parentheses necessary?